terraform/script.py

```diff
@@ -63,28 +59,76 @@ def get_envs():
     nodes_number = os.getenv("NODES_NUMBER")
     print(cluster_name)
 
-def confirm_config():
-    True
+
+def select_workspace():
+    global kube
+    if os.getenv("TF_VAR_CLOUDPROVIDER") == "aws":
+        kube = str("eks")
+    elif os.getenv("TF_VAR_CLOUDPROVIDER") == "gcp":
+        kube = str("gke")
+    elif os.getenv("TF_VAR_CLOUDPROVIDER") == "az":
+        kube = str("azkube")
+    
+    os.environ["TF_VAR_KUBE"] = kube
+
+        
+    output = str(subprocess.check_output("cd live/$TF_VAR_MQ/$TF_VAR_CLOUDPROVIDER/"+kube+" ; terragrunt workspace list", shell=True))
+    workspaces = output.replace("b'","").replace("'","").replace("\\n","").replace("*","").split()
+    
+    print("Select cluster to DELETE:")
+    check=False
+    while check == False:
+        count=0
+        for i in workspaces:
+            count+=1
+            print(str(count) + " - " + i)
+        
+        selected_workspace = int(input())
+        if selected_workspace in range(1, count+1):
+            print("Selected cluster: " + workspaces[int(selected_workspace) - 1])
+            check = True
+        else:
+            print("Select again proper value:")
```

Comment:
This is an interactive way of destroying a cluster. I think we should try to find another solution to allow it to be called in non-interactive way.

Reply:
Leaving as it is for now until new solution discussed.